### DISCUSSION TITLE
unable to start nginx service

### DISCUSSION THREAD

**Discussion by Mohandkajuluri** [learndevopsonline/learndevopsonline#1563] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/144593251/151e8db0-2f64-4cdb-b21a-81e540b95f2d)
Hi,
The above error is showing while running the nginx service, could you please address it.

**Comment by sreedharm07** [comment]
Hi @Mohandkajuluri 
please turn on your machine and ping your new public ip

**Comment by Mohandkajuluri** [comment]
Hi
Public id : 3.82.56.210

**Comment by sreedharm07** [comment]
hi @Mohandkajuluri 
remove space here after **http//** 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/0e9ea1d7-b430-41bc-b7e1-32247cdec596)

**Comment by Mohandkajuluri** [comment]
Got it.
Thank you.
